infra/gcp/ensure-conformance-storage.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to create the "k8s-conform" project and corresponding buckets in GCS.
+#
+# All conformance buckets are created in the "k8s-conform" GCP project, and are writable by
+# dedicated Google groups.
+#
+# The buckets will have a layout which is readable by testgrid:
+# (See also: https://github.com/kubernetes/test-infra/tree/master/gubernator#gcs-layout)
+# * each test result is stored in a "directory" with some monotonically increasing run identifier (commonly unix epoch time for third party uploads)
+# * they should contain a junit.*.xml file with the structured test results
+# * they should (optionally but commonly) contain a build-log.txt with the raw test output
+# * they should contain two small json files started.json and finished.json with some metadata
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+. "${SCRIPT_DIR}/lib.sh"
+
+function usage() {
+    echo "usage: $0 [repo...]" > /dev/stderr
+    echo "example:" > /dev/stderr
+    echo "  $0 # do all conformance repos" > /dev/stderr
+    echo "  $0 coredns # just do one" > /dev/stderr
+    echo > /dev/stderr
+}
+
+PROJECT="k8s-conform"
+
+PROJECT_VIEWER="k8s-infra-conform@kubernetes.io"
```

Comment:
I don't think we need this, do we?  What is it for?

Reply:
Seems to be a leftover from copying this script. I removed PROJECT_VIEWER